client/js/utils.js

```diff
@@ -141,3 +142,8 @@ function requestIdleCallback(callback, timeout) {
 		callback();
 	}
 }
+
+// Force handling preview display
+function togglePreviewMoreButtonsIfNeeded() {
+	window.requestAnimationFrame(() => $(window).trigger("resize"));
```

Comment:
Would it at least work if you triggered `resize` on current channel container instead of window?

Reply:
I'm not sure, but I'll try.

Reply:
I went a different route, I don't trigger `resize` anymore, and I can now target specifically current channel + link previews only + expanded previews only.
Will add more details in a comment in the main thread.